compiler/qsc_passes/src/capabilitiesck.rs

```diff
@@ -0,0 +1,538 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#[cfg(test)]
+mod base;
+
+#[cfg(test)]
+mod adaptive;
+
+#[cfg(test)]
+mod adaptive_plus_integers;
+
+#[cfg(test)]
+pub mod common;
```

Comment:
nitpick: could you prepend these with `tests_` so it's easier to see at a glance that they are not modules for production code?

Reply:
Modules renamed.